### ISSUE TITLE
.pyenv/libexec/pyenv line 43: cd: xxx.py: Not a directory

### ISSUE THREAD

**Issue by changsijay** [pyenv/pyenv#807]
$ cat test1.py
```
#!/usr/bin/env python
print("hello")
```

$ chmod +x test1.py
$ ln -s test1.py test2
$ ./test2

```
/home/csj/.pyenv/libexec/pyenv: line 43: cd: test1.py: Not a directory
hello
```

**Comment by jonganc** [comment]
Note: this is addressed (though not fixed/merge in #689 ). 
There, they point to a workaround: run `./configure && make` in pyenv's src dir, which will generate libexec/pyenv-realpath.dylib and work around the issue.

**Comment by blueyed** [comment]
Closing as duplicate of #580.
